consensus.go

```diff
@@ -46,3 +47,23 @@ func (c *Consensus) CommitState(state consensus.State) (consensus.State, error)
 	}
 	return c.actor.SetState(state)
 }
+
+// Subscribe returns a channel on which every new state is sent.
+func (c *Consensus) Subscribe() <-chan consensus.State {
+	c.chMux.Lock()
+	defer c.chMux.Unlock()
+	if c.subscriberCh == nil {
```

Comment:
This is because if the channel is nil, it means there are no subscribers so I don't put anything on the channel. Alternatively I'd have to throw a lock around a boolean to keep track if anyone is subscribed, and that's not pretty either... what do you think?